### PR TITLE
Rebrand ntp

### PR DESCRIPTION
**Pull request by ecotopian**

### CODE REVIEW

File: Client/Ecosia/UI/MyImpact/MyImpactCell.Indicator.swift [hunk 1]
```diff
@@ -33,11 +33,11 @@ extension MyImpactCell {
         
         required init?(coder: NSCoder) { nil }
         
-        init() {
-            super.init(frame: .zero)
+        init(size: CGSize) {
```
Comment (vauxhall) [inline Client/Ecosia/UI/MyImpact/MyImpactCell.Indicator.swift:36]: Great idea

File: Client/Ecosia/UI/MyImpact/MyImpactCell.Progress.swift [hunk 2]
```diff
@@ -18,20 +18,20 @@ extension MyImpactCell {
         
         required init?(coder: NSCoder) { nil }
         
-        init() {
-            super.init(frame: .zero)
+        init(size: CGSize, lineWidth: CGFloat) {
```
Comment (vauxhall) [inline Client/Ecosia/UI/MyImpact/MyImpactCell.Progress.swift:21]: Nice and clean

File: Client/Ecosia/UI/MyImpact/MyImpactCell.Indicator.swift [hunk 3]
```diff
@@ -9,8 +9,8 @@ extension MyImpactCell {
         var value = Double() {
             didSet {
                 let guide = CGMutablePath()
-                guide.addArc(center: .init(x: 120, y: 120),
-                             radius: 112,
+                guide.addArc(center: .init(x: bounds.size.width/2, y: bounds.size.width/2),
```
Comment (vauxhall) [inline Client/Ecosia/UI/MyImpact/MyImpactCell.Indicator.swift:12]: I like it (though I was under the impression NTP's counter didn't have `Indicator`, only `Progress`.)

**Review by vauxhall — APPROVED** [review]
Really nice

File: Client/Ecosia/UI/MyImpact/MyImpactCell.Indicator.swift (hunk 3 above)
Comment (ecotopian) [reply Client/Ecosia/UI/MyImpact/MyImpactCell.Indicator.swift:12]: That is true, it does not have the Indicator. I only noticed that later. Still I thought it's helpful that the Indicator is also configurable from the outside, don't you think?

Comment (vauxhall) [reply Client/Ecosia/UI/MyImpact/MyImpactCell.Indicator.swift:12]: We might never need to have a different configuration. But it is already done, so no harm on leaving it 👍

**Pull request merged**